frontend/messages/fr.json

```diff
@@ -592,11 +594,44 @@
 	"evidenceNoFile": "Aucun fichier n'a été téléchargé pour les preuves",
 	"requirementAppliedControlHelpText": "Les preuves liées aux mesures sélectionnées seront automatiquement associées à l'exigence.",
 	"requirementEvidenceHelpText": "Cet onglet vous permet d'ajouter des preuves supplémentaires à l'exigence.",
+	"providerID": "ID fournisseur",
+	"clientID": "ID client",
+	"secret": "Secrète",
+	"key": "Clé",
 	"settings": "Paramètres",
+	"identityProvider": "Fournisseur d'identité",
+	"identityProviders": "Fournisseurs d'identité",
+	"clientIDHelpText": "ID d'application ou clé consommateur",
+	"secretHelpText": "Secret API, secret client ou secret consommateur",
+	"SAMLIdPConfiguration": "Configuration IdP SAML",
+	"SPConfiguration": "Configuration SP",
 	"advancedSettings": "Réglages avancés",
+	"IdPEntityID": "ID d'entité IdP",
+	"metadataURL": "URL des métadonnées",
+	"SSOURL": "URL SSO",
+	"SLOURL": "URL SLO",
+	"x509Cert": "certificat x509",
+	"SPEntityID": "ID d'entité SP",
+	"attributeMappingUID": "Mappage d'attributs UID",
+	"attributeMappingEmailVerified": " ",
+	"allowRepeatAttributeName": "Autoriser la répétition du nom d'attribut",
+	"allowSingleLabelDomains": "Autoriser les domaines à étiquette unique",
+	"authnRequestSigned": "Demande d'authentification signée",
+	"digestAlgorithm": "Algorithme de synthèse",
+	"logoutRequestSigned": "Demande de déconnexion signée",
+	"logoutResponseSigned": "Réponse de déconnexion signée",
+	"metadataSigned": "Métadonnées signées",
+	"nameIDEncrypted": "Nom ID crypté",
```

Comment:
Some people are going to be **very** upset about this one. https://chiffrer.info/